pyproject.toml

```diff
@@ -2,6 +2,143 @@
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "mlflow"
+version = "2.10.3.dev0"
+description = "MLflow is an open source platform for the complete machine learning lifecycle"
+readme = "README.rst"
+keywords = ["mlflow", "ai", "databricks"]
+classifiers = [
+  "Intended Audience :: Developers",
+  "Programming Language :: Python :: 3.8",
+]
+requires-python = ">=3.8"
+dependencies = [
+  "click<9,>=7.0",
+  "cloudpickle<4",
+  "entrypoints<1",
+  "gitpython<4,>=3.1.9",
+  "pyyaml<7,>=5.1",
+  "protobuf<5,>=3.12.0",
+  "pytz<2024",
+  "requests<3,>=2.17.3",
+  "packaging<24",
+  "importlib_metadata<8,>=3.7.0,!=4.7.0",
+  "sqlparse<1,>=0.4.0",
+  "alembic<2,!=1.10.0",
+  "docker<8,>=4.0.0",
+  "Flask<4",
+  "numpy<2",
+  "scipy<2",
+  "pandas<3",
+  "querystring_parser<2",
+  "sqlalchemy<3,>=1.4.0",
+  "gunicorn<22; platform_system != 'Windows'",
+  "waitress<3; platform_system == 'Windows'",
+  "scikit-learn<2",
+  "pyarrow<16,>=4.0.0",
+  "markdown<4,>=3.3",
+  "Jinja2<4,>=2.11; platform_system != 'Windows'",
+  "Jinja2<4,>=3.0; platform_system == 'Windows'",
+  "matplotlib<4",
+  "graphene<4",
+]
+[[project.authors]]
+name = "Databricks"
+email = "databricks@databricks.com"
```

Comment:
This needs to be fixed.

Reply:
Let's update to maintainers email address

Reply:
sounds good!